compiler/src/dmd/target.d

```diff
@@ -196,7 +196,8 @@ void addPredefinedGlobalIdentifiers(const ref Target tgt)
             }
             case OS.OSX:
             {
-                predef("OSX");
+                predef("OSX");          // macOS
+                predef("Apple");        // macOS is one of Apple's operating systems
                 // For legacy compatibility
                 predef("darwin");
```

Comment:
interesting that this is there, yeah i do kinda recall the old days of D when it used this instead of OSX or whatever, but i forgot all about it.